### PR TITLE
Bump patch version, and update to latest ClimaTimeSteppers

### PR DESCRIPTION
**Pull request by charleskawczynski**
This PR bumps the ClimaCore patch version, and updates the manifest files.

### CODE REVIEW

File: examples/hybrid/sphere/hadley_circulation.jl [hunk 1]
```diff
@@ -235,10 +235,15 @@ parameters = (; coords, face_coords, τ, u₀, w₀, K, ρ₀, ystar, y_td)
 rhs!(ystar, y0, parameters, 0.0, dt, 1)
 
 # run!
-prob = ODEProblem(IncrementingODEFunction(rhs!), copy(y0), (0.0, T), parameters)
+prob = ODEProblem(
+    ClimaODEFunction(; T_exp != rhs!),
```
Comment (valeriabarra) [inline examples/hybrid/sphere/hadley_circulation.jl:239]:
```suggestion
    ClimaODEFunction(; T_exp! = rhs!),
```
I think the white space was misplaced here!

**Comment by simonbyrne** [comment]
bors try

File: Project.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name = "ClimaCore"
 uuid = "d414da3d-4745-48bb-8d80-42e94e092884"
 authors = ["CliMA Contributors <clima-software@caltech.edu>"]
-version = "0.10.25"
+version = "0.10.26"
```
Comment (simonbyrne) [inline Project.toml:4]: Given that we've broken a bunch of stuff, we may need to make the a breaking release.

**Comment by dennisYatunin** [comment]
bors try

**Pull request merged**